### PR TITLE
Added stable APIs for release SDK

### PR DESCRIPTION
**Pull request by nishitha-burman**
Prerelease coming soon

AB#42997924

### CODE REVIEW

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -37,6 +37,179 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
 <!-- ====================================================================== -->
 ## 1.0.1462.37
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:38]:
```suggestion
## 1.0.1518.46
```
https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1518.46

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -34,6 +34,179 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2EnvironmentOptions3::IsCustomCrashReportingEnabled property (get](/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions3?view=webview2-1.0.1518.46&preserve-view=true#get_iscustomcrashreportingenabled), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions3?view=webview2-1.0.1518.46&preserve-view=true#put_iscustomcrashreportingenabled)
+* [ICoreWebView2Environment11::FailureReportFolderPath property (get)](/microsoft-edge/webview2/reference/win32/icorewebview2environment11?view=webview2-1.0.1518.46&preserve-view=true#get_failurereportfolderpath)<!--no put-->
+
+---
+
+<!-- ====================================================================== -->
+
+## Prereleasae coming soon
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:125]:
```suggestion
## Prerelease coming soon
```

File: microsoft-edge/webview2/release-notes.md [hunk 3]
```diff
@@ -34,6 +34,179 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+* [ICoreWebView2PrintCompletedHandler interface](/microsoft-edge/webview2/reference/win32/icorewebview2printcompletedhandler?view=webview2-1.0.1518.46&preserve-view=true)
+* [ICoreWebView2PrintToPdfStreamCompletedHandler interface](/microsoft-edge/webview2/reference/win32/icorewebview2printtopdfstreamcompletedhandler?view=webview2-1.0.1518.46&preserve-view=true)
+* [ICoreWebView2PrintSettings2 interface](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1549-prerelease&preserve-view=true)
+   * [ICoreWebView2PrintSettings2::Collation property (get](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#get_collation), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#put_collation)
+   * [ICoreWebView2PrintSettings2::ColorMode property (get](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1549-prerelease&preserve-view=true#get_colormode), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#put_colormode)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:94]: Global: some links say 1546, some say 1549.  source of truth:
[NuGet package for WebView2 SDK 1.0.1518.46](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1518.46)

Make them all say 1.0.1518.46?

File: microsoft-edge/webview2/release-notes.md (hunk 2 above)
Comment (nishitha-burman) [reply microsoft-edge/webview2/release-notes.md:125]: We can remove this line until we have the prerelease details

File: microsoft-edge/webview2/release-notes.md (hunk 3 above)
Comment (nishitha-burman) [reply microsoft-edge/webview2/release-notes.md:94]: Correct, the versions should be 1.0.1518.46

**Review by JasonAndrewWriter — APPROVED** [review]

File: microsoft-edge/webview2/release-notes.md [hunk 4]
```diff
@@ -34,6 +34,179 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+   * [CoreWebView2PrintSettings.MediaSize Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2printsettings?view=webview2-winrt-1.0.1518.46&preserve-view=true#mediasize)
+   * [CoreWebView2PrintSettings.PageRanges Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2printsettings?view=webview2-winrt-1.0.1518.46&preserve-view=true#pageranges)
+   * [CoreWebView2PrintSettings.PagesPerSide Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2printsettings?view=webview2-winrt-1.0.1518.46&preserve-view=true#pagesperside)
+   * [CoreWebView2PrintSettings.PrinterName Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2printsettings?view=webview2-winrt-1.0.1518.46&preserve-view=true#printername)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2_16 interface](/microsoft-edge/webview2/reference/win32/icorewebview2_16?view=webview2-1.0.1549-prerelease&preserve-view=true)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:86]: Global in this Win32 list: find/fix "prerelease" in links
```suggestion
* [ICoreWebView2_16 interface](/microsoft-edge/webview2/reference/win32/icorewebview2_16?view=webview2-1.0.1518.46&preserve-view=true)
```
Based on live links below, the correct ?view strings for this h2 section are:

.NET:
?view=webview2-dotnet-1.0.1518.46

WinRT:
?view=webview2-winrt-1.0.1518.46

Win32:
?view=webview2-1.0.1518.46

Actual live URLs:

.NET:
https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2?view=webview2-dotnet-1.0.1518.46

WinRT:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=webview2-winrt-1.0.1518.46

Win32:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2?view=webview2-1.0.1518.46

https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1518.46

File: microsoft-edge/webview2/release-notes.md [hunk 5]
```diff
@@ -34,6 +34,179 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2_16 interface](/microsoft-edge/webview2/reference/win32/icorewebview2_16?view=webview2-1.0.1518.46&preserve-view=true)
+   * [ICoreWebView2_16::Print method](/microsoft-edge/webview2/reference/win32/icorewebview2_16?view=webview2-1.0.1518.46&preserve-view=true#print)
+   * [ICoreWebView2_16::PrintToPdfStream method](/microsoft-edge/webview2/reference/win32/icorewebview2_16?view=webview2-1.0.1518.46&preserve-view=true#printtopdfstream)
+   * [ICoreWebView2_16::ShowPrintUI method](/microsoft-edge/webview2/reference/win32/icorewebview2_16?view=webview2-1.0.1518.46&preserve-view=true#showprintui)
+* [ICoreWebView2PrintCompletedHandler interface](/microsoft-edge/webview2/reference/win32/icorewebview2printcompletedhandler?view=webview2-1.0.1518.46&preserve-view=true)
+* [ICoreWebView2PrintToPdfStreamCompletedHandler interface](/microsoft-edge/webview2/reference/win32/icorewebview2printtopdfstreamcompletedhandler?view=webview2-1.0.1518.46&preserve-view=true)
+* [ICoreWebView2PrintSettings2 interface](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1549-prerelease&preserve-view=true)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:92]:
```suggestion
* [ICoreWebView2PrintSettings2 interface](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true)
```

File: microsoft-edge/webview2/release-notes.md [hunk 6]
```diff
@@ -34,6 +34,179 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+   * [ICoreWebView2PrintSettings2::Collation property (get](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#get_collation), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#put_collation)
+   * [ICoreWebView2PrintSettings2::ColorMode property (get](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1549-prerelease&preserve-view=true#get_colormode), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#put_colormode)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:94]:
```suggestion
   * [ICoreWebView2PrintSettings2::ColorMode property (get](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1549-prerelease&preserve-view=true#get_colormode), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#put_colormode)
```
win32

Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:94]:
```suggestion
   * [ICoreWebView2PrintSettings2::ColorMode property (get](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#get_colormode), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#put_colormode)
```

File: microsoft-edge/webview2/release-notes.md [hunk 7]
```diff
@@ -34,6 +34,179 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+   * [ICoreWebView2PrintSettings2::Copies property (get](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1549-prerelease&preserve-view=true#get_copies), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#put_copies)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:95]:
```suggestion
   * [ICoreWebView2PrintSettings2::Copies property (get](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#get_copies), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#put_copies)
```

File: microsoft-edge/webview2/release-notes.md [hunk 8]
```diff
@@ -34,6 +34,179 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+   * [ICoreWebView2PrintSettings2::Duplex property (get](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1549-prerelease&preserve-view=true#get_duplex), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#put_duplex)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:96]:
```suggestion
   * [ICoreWebView2PrintSettings2::Duplex property (get](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#get_duplex), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#put_duplex)
```

File: microsoft-edge/webview2/release-notes.md [hunk 9]
```diff
@@ -34,6 +34,179 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+   * [ICoreWebView2PrintSettings2::MediaSize property (get](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1549-prerelease&preserve-view=true#get_mediasize), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#put_mediasize)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:97]:
```suggestion
   * [ICoreWebView2PrintSettings2::MediaSize property (get](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#get_mediasize), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#put_mediasize)
```

File: microsoft-edge/webview2/release-notes.md [hunk 10]
```diff
@@ -34,6 +34,179 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+   * [ICoreWebView2PrintSettings2::PageRanges property (get](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1549-prerelease&preserve-view=true#get_pageranges), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#put_pageranges)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:98]:
```suggestion
   * [ICoreWebView2PrintSettings2::PageRanges property (get](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#get_pageranges), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#put_pageranges)
```

File: microsoft-edge/webview2/release-notes.md [hunk 11]
```diff
@@ -34,6 +34,179 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+   * [ICoreWebView2PrintSettings2::PagesPerSide property (get](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1549-prerelease&preserve-view=true#get_pagesperside), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#put_pagesperside)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:99]:
```suggestion
   * [ICoreWebView2PrintSettings2::PagesPerSide property (get](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#get_pagesperside), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2?view=webview2-1.0.1518.46&preserve-view=true#put_pagesperside)
```

File: microsoft-edge/webview2/release-notes.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md:125]: @nishitha-burman - In my Suggestion markup, removed the line entirely

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**